apel/db/records/cloud.py

```diff
@@ -38,25 +38,25 @@ def __init__(self):
         self._mandatory_fields = ["VMUUID", "SiteName"]
             
         # This list allows us to specify the order of lines when we construct records.
-        self._msg_fields  = ["VMUUID", "SiteName", "MachineName", 
+        self._msg_fields  = ["VMUUID", "SiteName", "CloudComputeService", "MachineName", 
                              "LocalUserId", "LocalGroupId", "GlobalUserName", "FQAN",
                              "Status", "StartTime", "EndTime", "SuspendDuration", 
-                              "WallDuration",
-                             "CpuDuration", "CpuCount", "NetworkType", "NetworkInbound", 
-                             "NetworkOutbound",
-                             "Memory", "Disk", "StorageRecordId", "ImageId", "CloudType"]
+                             "WallDuration", "CpuDuration", "CpuCount", 
+                             "NetworkType", "NetworkInbound", "NetworkOutbound", "PublicIPCount", 
+                             "Memory", "Disk", "BenchmarkType", "Benchmark", 
+                             "StorageRecordId", "ImageId", "CloudType"]
         
         # This list specifies the information that goes in the database.
-        self._db_fields = self._msg_fields[:7] + ['VO', 'VOGroup', 'VORole'] + self._msg_fields[7:]
+        self._db_fields = self._msg_fields[:8] + ['VO', 'VOGroup', 'VORole'] + self._msg_fields[8:]

```

Comment:
yes, because FQAN has moved from index 7 to 8 with the addition of `CloudComputeService` so changing `7` to `8` at line 50 means the VO fields stay in the same place